docs/concepts/controlling-catalog-selection.md

```diff
@@ -18,7 +18,7 @@ To select a specific catalog by name, you can use the `matchLabels` field in you
 #### Example
 
 ```yaml
-apiVersion: olm.operatorframework.io/v1alpha1
+apiVersion: olm.operatorframework.io/v1
 kind: ClusterExtension
```

Comment:
Watch out - this is `ClusterExtension` and we still don't have v1 for it. This comment applies to other places.

Reply:
thank you!